public/app/features/explore/Wrapper.tsx

```diff
@@ -26,10 +26,8 @@ const styles = {
     width: 100%;
     flex-grow: 1;
     min-height: 0;
-  `,
-  exploreWrapper: css`
-    display: flex;
     height: 100%;
+    position: relative;
```

Comment:
Why `position: relative` is needed here?

Reply:
@ifrost the SplitPaneWrappers create absolutely positioned containers, and it should be relative to this outer wrapper